setup.examples/exim/mailwatch-exim.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+# Script to apply adjustments for exim
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+Webuser="$1"
+
+/etc/init.d/exim4 stop
```

Comment:
Fedora / RHEL / CentOS Linux 

In fact RedHad started to use "service" many years before Debian.

https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/System_Administrators_Guide/sect-Managing_Services_with_systemd-Services.html